src/components/MAPI/organizations/AccessControl/AccessControlRoleSubjects.tsx

```diff
@@ -155,7 +184,10 @@ interface IAccessControlRoleSubjectsProps
   roleName: string;
   namespace: string;
   permissions: ui.IAccessControlPermissions;
-  onAdd: (type: ui.AccessControlSubjectTypes, names: string[]) => Promise<void>;
+  onAdd: (
+    type: ui.AccessControlSubjectTypes,
+    names: string[]
+  ) => Promise<void | ui.AccessControlRoleSubjectStatus[]>;
```

Comment:
Is it required to still allow `Promise<void>`? When do we resolve without a value?